app/components/Repos/Card/styledComponents.js

```diff
@@ -18,17 +18,21 @@ const { mobile } = mediaQueriesByDevice;
 
 export const ContentContainer = styled.div`
```

Comment:
Also, let's add `margin-left: 1rem;` here so that there is a bit more spacing between the image and content
<img width="411" alt="Screen Shot 2021-05-21 at 12 50 43 AM" src="https://user-images.githubusercontent.com/44502862/119083771-976f7e00-b9ce-11eb-8139-4af2b4d45c53.png">